general/summary/cli.go

```diff
@@ -71,6 +80,26 @@ func GenerateSummaryMarkdown(c *cli.Context) error {
 	return saveMarkdownToFileSystem(finalMarkdown)
 }
 
+func aggregatedCodeScanningSarifs() error {
+	files, err := getSarifFiles()
```

Comment:
I've written a utility function that does that:

`commandSummary.GetIndexedDataFilesPaths()`

It will get you a map of saved files under all the indexes,
you can then filter only the relevant "sarif" index .

The returned value is a list of file paths to all the files under the index.